src/css/mediaelementplayer-legacy.css

```diff
@@ -617,17 +603,12 @@ Reference: http://blog.rrwd.nl/2015/04/04/the-screen-reader-text-class-why-and-h
 
 .mejs-captions-selector-input,
 .mejs-chapters-selector-input {
-    clear: both;
-    float: left;
-    left: -1000px;
-    margin: 3px 3px 0 5px;
-    position: absolute;
+    display: none;
```

Comment:
We can't use this because it affects accessibility when users tab to select captions/chapters. This must be reverted to the way it was